### DISCUSSION TITLE
Change dynamically the path from service_account_credentials_json

### DISCUSSION THREAD

**Discussion by paleomariomm** [freekmurze/laravel-analytics#441] [General]
We are using the `spatie/laravel-analytics` [Github Package](https://github.com/spatie/laravel-analytics) to get data from Google Analytics and create graphs.

We need to know if in the package file `spatie/laravel-analytics/config/analytics.php`, the variable `service_account_credentials_json` can only be the path to a single file or can it be changed dynamically per account.

We would like to be able to update this parameter so that it is not fixed in the same way as the `view_id field` can be updated using the `SetViewId()` function.

Would this be possible? If so, how could we do it?

**Comment by paleomariomm** [comment]
Hi, any idea on the question I sent some days ago? I would appreciate an answer.

Best

**Comment by mrtsglk** [comment]
I'm having the same problem. Is there any solution for this?
